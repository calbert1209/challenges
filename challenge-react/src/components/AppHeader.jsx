import React from 'react';
import styled from 'styled-components';

const HeaderStyled = styled.header`
  text-align: center;
`;

const AppHeader = () => {
  return (
    <HeaderStyled>
      <h1>Omise Tamboon React</h1>
    </HeaderStyled>
  );
};

export default AppHeader;
