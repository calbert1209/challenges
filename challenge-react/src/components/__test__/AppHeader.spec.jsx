import React from 'react'
import TestRenderer from 'react-test-renderer';
import AppHeader from '../AppHeader';
describe('AppHeader', () => {
  it('render correctly', () => {
    const testRenderer = TestRenderer.create(<AppHeader />);

    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
