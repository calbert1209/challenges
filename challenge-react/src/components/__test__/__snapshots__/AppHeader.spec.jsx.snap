// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader render correctly 1`] = `
<header
  className="sc-bdfBwQ hMdVtH"
>
  <h1>
    Omise Tamboon React
  </h1>
</header>
`;
